### DISCUSSION TITLE
Discord-Bot: Mod-Updates

### DISCUSSION THREAD

**Discussion by LimitlessDrag0n** [gameserverapp/Platform#552] [Suggestions]
It would be cool, if there where an option for all server to get a "Mod X has an update. Will be updated soon!" to Discord. 

![image](https://github.com/gameserverapp/Platform/assets/99832504/2cc3b8fb-3442-47c2-8bfb-003674da5e33)

Maybe a checkmark for that under those already there?

**Comment by LimitlessDrag0n** [comment]
Yes, the severstatus card shows updates but all other statusses as well. 
![image](https://github.com/gameserverapp/Platform/assets/99832504/8a296158-41fa-4db9-83f2-48569bd8b43d)
My thought was on just the mod update information so people can see just that information

**Comment by birdman3131** [comment]
https://steam.watch/ is a discord bot that can notify you of updates on steam workshop items if you want.

**Reply by LimitlessDrag0n** [reply]
Yeah I know about that but having multiple servers, different games and sometimes just testing switching mods remove mods etc, it would be nice to have a feature on the tool, I'm already having if they already have a similar feature

**Comment by mvaessen** [comment]
This should already be available & working
